WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -228,7 +244,10 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
             segmentedControl.centerXAnchor.constraint(equalTo: segmentedControlContainerView.centerXAnchor),
             segmentedControl.topAnchor.constraint(equalTo: segmentedControlContainerView.topAnchor,
                                                   constant: Constants.segmentedControlYOffset),
-            segmentedControl.bottomAnchor.constraint(equalTo: segmentedControlContainerView.bottomAnchor)
+            segmentedControl.bottomAnchor.constraint(equalTo: segmentedControlContainerView.bottomAnchor),
+            segmentedControl.heightAnchor.constraint(equalToConstant: Constants.segmentedControlHeight),
+            siteMenuSpotlightView.leadingAnchor.constraint(equalTo: segmentedControl.leadingAnchor, constant: -Constants.siteMenuSpotlightOffset),
```

Comment:
```suggestion
            
            // Quick start spotlight
            siteMenuSpotlightView.leadingAnchor.constraint(equalTo: segmentedControl.leadingAnchor, constant: -Constants.siteMenuSpotlightOffset),
```

The number of constraints are getting bigger, this can help to understand what's what (or maybe extracting to different methods).

Reply:
Fixed! 418cf60